jmeter/build.gradle

```diff
@@ -82,9 +91,24 @@ task pluginJar(type: Jar, dependsOn: compileJava) {
     include '**/*.class'
 }
 
+assemble.dependsOn pluginJar
+assemble.dependsOn pluginSupportJar
+
+compileJava {
+    doLast {
```

Comment:
i know weve merged but - we should probably get travis build binaries anyway: in `.travis.yml` add `script: gradle build`